Sources/Widgets/Widgets3D/ResliceCursorWidget/index.js

```diff
@@ -267,6 +263,12 @@ function vtkResliceCursorWidget(publicAPI, model) {
     renderer.getActiveCamera().setFocalPoint(...estimatedFocalPoint);
     renderer.getActiveCamera().setPosition(...estimatedCameraPosition);
 
+    // Update view up in sagittal view
+    if (viewType === ViewTypes.SAGITTAL) {
+      const viewUp = [0, 0, 1];
+      renderer.getActiveCamera().setViewUp(viewUp);
```

Comment:
can't it be done in `computeReslicePlaneOrigin()` ?

Reply:
We don't have access to camera in this method. Do you want to return an object ?

Reply:
I wonder if the same end result as setting a view up can be achieved by changing what "point1" and "point2" mean.

Reply:
Okay! Just rechecked the point0/1/2 set in the planeSource.
There are already well defined:
0 = bottom left
1 = bottom right
2 = top left
So that we can compute the view up thanks to points 0 and 1